test/framework/kogitoserviceutils.go

```diff
@@ -122,7 +145,11 @@ func cliInstall(service v1alpha1.KogitoService, cliName string, cliFlags []strin
 		cmd = append(cmd, cliFlag)
 	}
 
-	cmd = append(cmd, "--image", framework.ConvertImageToImageTag(*service.GetSpec().GetImage()))
+	image := framework.ConvertImageToImageTag(*service.GetSpec().GetImage())
```

Comment:
This change was not necessary, but it was weird to send only "--image" with a value. In fact, The operator is behaving differently when sending "--image" (without value) and when not sending the "--image". I raised a defect for this: https://issues.redhat.com/browse/KOGITO-1961

Reply:
good for me